ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.scss

```diff
@@ -0,0 +1,308 @@
+.details-header {
```

Comment:
Maybe it would be better to add these styles to the general scss. From what I understand we would like to unify detail pages of trainings and guidelines.

Reply:
I didn't remove css before code review. done